src/components/Checkbox/__tests__/Checkbox.spec.js

```diff
@@ -6,7 +6,6 @@ const initialProps = {
   modelValue: false,
   name: 'Test name',
   label: 'Test',
-  error: false,
```

Comment:
isn't there error state anymore? I think it's useful when a checkbox is required

Reply:
hmm it was not in the design and we do not have error on any checkbox that we use!
we went over this with Michael and did not see a need for it, can you point a case where a checkbox should be red?
my only thought is a 'I agree to terms and conditions' type of checkbox but we have that and we do not mark it as required, the signup form does its own check and shows an alert